docs/USER_RECORD.md

```diff
@@ -597,6 +597,17 @@ The salt to pass to the FIDO2 device is found in `fido2HmacSalt`.
 The only supported recovery key type at the moment is `modhex64`, for details see the description of `recoveryKey` below.
 An account may have any number of recovery keys defined, and the array should have one entry for each.
 
+`selfModifiableFields` → An array of strings, each cooresponding to a field name that can appear
+in the `regular` or `perMachine` sections. The user may be allowed to edit any field in this list
+without authenticating as an administrator. Note that the user will only be allowed to edit fields
+in `perMachine` sections that match the machine the user is performing the edit from.
+
+`selfModifiableBlobs` → Similar to `selfMoidifiableFields`, but it lists blobs that the user
+is allowed to edit.
+
+`selfModifiablePrivileged` →  Similar to `selfMoidifiableFields`, but it lists fields in
+the `privileged` section that the user is allowed to edit.
```

Comment:
why keep this separate from "regular" and "perMachine"? i mean, we'd be really stupid if we'd introduce fields in one section that would exist (or if existance then with different meaning) in the other. Hence I think it should be fine to throw them in the same bin here and keep one list of json fields.

Reply:
The spec places no such restriction on the JSON. So it's entirely valid right now for some provider of user records to add identical custom fields with different semantics across the two sections.

We can probably get away with changing the spec right now, since ostensibly we're the only consumers of it and the change would be backwards-compatible for us. But it's something we have to explicitly document, or just handle here.